icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -14,7 +14,16 @@
 #include "unicode/uniset.h"
 #include "unicode/uscript.h"
 #include "unicode/putil.h"
+#include "unicode/uchar.h"
 #include "unicode/umutablecptrie.h"
+#include "unicode/uniset.h"
+#include "charstr.h"
+#include "cmemory.h"
+#include "cstring.h"
+#include "toolutil.h"
+#include "ucase.h"
+#include "udatamem.h"
```

Comment:
Please remove this line. Sorry if I missed that earlier.

Reply:
The force-push shows that you kept this line but removed the line for "unicode/umutablecptrie.h" instead. You do need that one. Please fix.

Reply:
Oh, oops, I misread your comment. Strangely, it built just fine without the umutablecptrie.h include; I guess it must be transitively included somewhere. 

(Upon review: writesrc.h)

Looking closer, it appears that I messed up the merge with the scx changes. I'll clean that up.